src/api/apiConfig.ts

```diff
@@ -1,7 +1,7 @@
 import { Configuration } from 'open-bus-stride-client'
 
 //const BASE_PATH = 'http://localhost:3000/api'
-export const BASE_PATH = import.meta.env.VITE_BASE_PATH
+export const BASE_PATH = process.env.VITE_BASE_PATH
```

Comment:
this will fix the base href issue? #323 
I guess we should / could add a Playwright test for that

Reply:
> may i ask - do you have a discord account?

feel free to add me! my username is ```camille.kh```